addons/addon-environment-sc-api/packages/environment-sc-workflow-steps/lib/steps/check-launch-dependency/check-launch-dependency.js

```diff
@@ -64,370 +64,386 @@ const successStatuses = ['CREATE_COMPLETE', 'DELETE_COMPLETE', 'UPDATE_COMPLETE'
  * Dependency check is performed based on the output of the product CFT
  */
 class CheckLaunchDependency extends StepBase {
```

Comment:
Will you be adding tests for this class?

Reply:
Yes. You can expect unit test cases in the subsequent PRs